### ISSUE TITLE
Occasional no highlight in show_prediction for images

### ISSUE THREAD

**Issue by abedkhooli** [TeamHG-Memex/eli5#368]
I use ktrain wrapper library for Keras which calls 
`eli5.show_prediction(model, x)` [eli5: '0.10.1']
and for some images, there is no indication (color highlight) on the predicted image (both correct and incorrect classification). Ex:
![image](https://user-images.githubusercontent.com/11407254/76288878-8b5b3d00-62b0-11ea-8920-c7eded79a9ef.png)

**Comment by teabolt** [comment]
HI @abedkhooli,

Could you post the result of `eli5.explain_prediction(model, x)`? This should show if it's a problem with the explanation or with the color highlight.

You can also try changing the color highlight like this: `eli5.show_prediction(model, x, alpha_limit=1.0, colormap=matplotlib.cm.cividis)`

**Comment by abedkhooli** [comment]
Ktrain uses a forked version with tf.keras and TF2.x, so these won't work. I tried with one less epoch and got it highlighted correctly! 
![image](https://user-images.githubusercontent.com/11407254/76448336-da998e80-63d2-11ea-9157-6b2325637edd.png)
